TimeSeriesChart/cue.mod/module.cue

```diff
@@ -1,4 +1,4 @@
 module: "github.com/perses/plugins/timeserieschart@v0"
 language: {
-	version: "v0.11.0"
+	version: "v0.12.0"
```

Comment:
probably we should bump the version for all plugin

Reply:
cue version upgrade for all plugins to be tackled in a separate PR as agreed